workflows/sanger-variant-calling/jtracker/workflow/main.jt

```diff
@@ -0,0 +1,508 @@
+# This workflow
+
+workflow:
+  name:  sanger-variant-calling
+  version: "0.1.0"
+
+  input:
+    song_collab_url:
+      type: string
+      default: "https://song.cancercollaboratory.org"
+    song_aws_url:
+      type: string
+      default: "https://virginia.song.icgc.org"
+    reference:
+      type: string
+      is_file: true
+      default: "[reference]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/core_ref_GRCh38_hla_decoy_ebv.tar.gz"
+    annot:
+      type: string
+      is_file: true
+      default: "[annot]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/VAGrENT_ref_GRCh38_hla_decoy_ebv_ensembl_91.tar.gz"
+    snv_indel:
+      type: string
+      is_file: true
+      default: "[snv_indel]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/SNV_INDEL_ref_GRCh38_hla_decoy_ebv-fragment.tar.gz"
+    cnv_sv:
+      type: string
+      is_file: true
+      default: "[cnv_sv]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/CNV_SV_ref_GRCh38_hla_decoy_ebv_brass6+.tar.gz"
+    qcset:
+      type: string
+      is_file: true
+      default: "[qcset]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/qcGenotype_GRCh38_hla_decoy_ebv.tar.gz"
```

Comment:
change `[qcset]` to`[qcGenotype_GRCh38_hla_decoy_ebv.tar.gz]`

Reply:
done